### PR TITLE
Cypress/FE-2160 Checkbox and RadioButton validations components

### PR DESCRIPTION
**Pull request by KatarzynaQA**
# Description
Added tests for checkbox and radio button validations components

### CODE REVIEW

File: cypress/features/regression/validations/checkboxValidations.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  I want to change Checkbox validations component properties
+
+  Background: Open Checkbox validations component page
+    Given I open "Experimental Checkbox" component page validations in iframe
+
+  @positive
+  @validations
+  Scenario Outline: Verify the <state> validation of Checkbox in form component
+    Given I check "<position>" checkbox 2 times
+    When I hover mouse onto "<state>" icon into iFrame
```
Comment (DlgSHi) [inline cypress/features/regression/validations/checkboxValidations.feature]: We could use more proper sentence `icon in iFrame`

File: cypress/support/step-definitions/checkbox-steps.js [hunk 2]
```diff
@@ -37,3 +44,59 @@ Then('Checkbox label width is set to {int}', (width) => {
...
+    case 'fifth':
+      checkbox(FIFTH_CHECKBOX).click();
+      break;
+    case 'sixth':
+      checkbox(SIXTH_CHECKBOX).click();
+      break;
+    case 'seventh':
+      checkbox(SEVENTH_CHECKBOX).click();
+      break;
+    default: throw new Error('There are only seven checkboxes elements on the page');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/checkbox-steps.js]: `checkbox elements`

File: cypress/support/step-definitions/checkbox-steps.js [hunk 3]
```diff
@@ -37,3 +44,59 @@ Then('Checkbox label width is set to {int}', (width) => {
...
+      case 'fifth':
+        checkbox(FIFTH_CHECKBOX, times).click();
+        break;
+      case 'sixth':
+        checkbox(SIXTH_CHECKBOX, times).click();
+        break;
+      case 'seventh':
+        checkbox(SEVENTH_CHECKBOX, times).click();
+        break;
+      default: throw new Error('There are only seven checkboxes elements on the page');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/checkbox-steps.js]: `checkbox elements` or `checkboxes`

File: cypress/support/step-definitions/common-steps.js [hunk 4]
```diff
@@ -146,6 +153,33 @@ Then('I hover mouse onto {string} icon into iFrame', (name) => {
   getElementNoIframe(name).trigger('mouseover');
 });
 
+Then('I hover mouse onto {string} {string} icon for validations component into iFrame', (position, name) => {
```
Comment (DlgSHi) [inline cypress/support/step-definitions/common-steps.js]: probably we should move this step to Validation-steps ?

File: cypress/support/step-definitions/common-steps.js [hunk 5]
```diff
@@ -146,6 +153,33 @@ Then('I hover mouse onto {string} icon into iFrame', (name) => {
...
+    case 'fifth':
+      getDataElementByValueNoIframe(name).eq(FIFTH_ELEMENT).trigger('mouseover');
+      break;
+    case 'sixth':
+      getDataElementByValueNoIframe(name).eq(SIXTH_ELEMENT).trigger('mouseover');
+      break;
+    case 'seventh':
+      getDataElementByValueNoIframe(name).eq(SEVENTH_ELEMENT).trigger('mouseover');
+      break;
+    default: throw new Error('There are only seven validation icons elements on the page');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/common-steps.js]: `validation icons` or `validation icon elements`

File: cypress/support/step-definitions/radio-button-steps.js [hunk 6]
```diff
@@ -9,3 +13,18 @@ Then('fieldHelpInline is enabled', () => {
...
+    case 'first':
+      radioButton().eq(FIRST_ELEMENT).click();
+      break;
+    case 'second':
+      radioButton().eq(SECOND_ELEMENT).click();
+      break;
+    case 'third':
+      radioButton().eq(THIRD_ELEMENT).click();
+      break;
+    default: throw new Error('There are only three validation icons elements on the page');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/radio-button-steps.js]: `validation icons` or `validation icon elements`

Comment (KatarzynaQA) [reply cypress/support/step-definitions/radio-button-steps.js]: fixed

File: cypress/support/step-definitions/common-steps.js (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/common-steps.js]: fixed

File: cypress/support/step-definitions/checkbox-steps.js (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/checkbox-steps.js]: fixed

File: cypress/support/step-definitions/checkbox-steps.js (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/checkbox-steps.js]: fixed

File: cypress/features/regression/validations/checkboxValidations.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/validations/checkboxValidations.feature]: fixed

File: cypress/support/step-definitions/common-steps.js (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/common-steps.js]: This step is used in two different components, that's the reason why I put it here --> checkbox and validations component.

**Pull request merged**